src/components/Person/Person.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+function Person({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) {
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {
+        age
+          ? (
+            <p className="Person__age">
```

Comment:
use `&&`

Reply:
Actually I tried but then it prints `0` on the page for an unknown reason to me :)

Reply:
it prints zero because you have set 0 to be the default prop. do not make it default, just make it not required